content.js

```diff
@@ -21,18 +21,24 @@ function copyPrDescription() {
   messageField.value = commitBody;
 }
 
-function addMergeListeners() {
-  const squashButton = document.querySelector('.merge-message .btn-group-squash');
-  const mergeButton = document.querySelector('.merge-message .btn-group-merge');
+function addMergeListener(event) {
+  if (!window.location.pathname.match("/pull/[0-9]+$")) return;
```

Comment:
Maybe this should gate everything? I'm not sure exactly when content.js gets loaded/run, but I'm guessing it is re-invoked when the location changes?

Something like:


```
function main() {
  if (!window.location.pathname.match("/pull/[0-9]+$")) return;

  // Run on extension load.
  addMergeListener();

  // Run on AJAX.
  document.addEventListener('pjax:end', addMergeListener);

  // Run when new comments are added, removed, edited, etc.
  const comments = document.querySelector('.js-discussion');
  if (comments) {
    const observer = new MutationObserver(addMergeListener);
    observer.observe(comments, {childList: true});
  }
}

main();
```

WDYT?

Reply:
Testing locally this seems to still work for me